spacy_loggers/mlflow.py

```diff
@@ -11,6 +11,13 @@
 from spacy.training.loggers import console_logger
 
 
+class ModelDir:
+    def __init__(self) -> None:
+        self.path = None
+
+    def update(self, path: str) -> None:
+        self.path = path
+        
```

Comment:
I think we can ditch `ModelDir` - let's use a simple string for the latest model path (`latest_model_dir` or similar).